src/algebraic_topology/simplicial_object.lean

```diff
@@ -211,8 +211,8 @@ def whiskering_obj (D : Type*) [category D] (F : C ⥤ D) :
     w' := begin
       ext,
       dsimp,
-      erw [category.comp_id, category.comp_id, ← F.map_comp,
-        ← F.map_comp, ← nat_trans.comp_app, η.w],
+      rw [category.comp_id, category.comp_id, ← F.map_comp, ← F.map_comp, ← nat_trans.comp_app],
+      erw η.w,
```

Comment:
I wonder if we should have two variants of `comma_morphism.w` for when one of the functors in the comma is the identity.